spec/factories/meeting_option_factory.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryBot.define do
   factory :meeting_option do
-    name { Faker::Vehicle.unique.make }
+    name { Faker::Address.unique.street_address }
```

Comment:
Faker was throwing an error.
There is not enough `Vehicule.make`, which is a problem because of the unique tag.
I double checked and it doesn't seem like we are creating a crazy amount of meeting_options.

Reply:
@KH-Amir-TN has suggested to clear the cache.
It seems like the uniques stored in cache does not get cleared in between spec groups.
`Vehicle.unique.make` is also used in `room_settings_getter_spec.rb`

Reply:
I think using Lorem.word is more adequate than Vehicle.make and should solve are our unique scarcity issues.